npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -143,8 +150,7 @@ def path_dependency_details(fetched_files)
           package_json_path_deps +=
             path_dependency_details_from_manifest(file)
         end
-
-        path_starts = %w(file: link:.)
+        path_starts = NPM_PATH_DEPENDENCY_STARTS
```

Comment:
Pretty sure this is correct, npm bails out if you try and install dependencies with `link:`: `Unsupported URL Type "link"`:   https://npm.community/t/is-link-or-file-supported/1577

Reply:
Makes sense. Might be nice to split these out into named methods (`path_dependency_details_from_npm_lockfile` etc.)

Reply:
Yeah nice!